### DISCUSSION TITLE
[Question/Idea] Combo counter animation

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#21180] [Q&A] [answered]
Hi, came ghere to ask if there's any way to return the old animation to the counter in lazer.
Back in time, changing the origin of the object would make the animation also move to the origin which sadly isn't possible anymore apparently, because of that the animation now comes from the top-right side of the counter instead of  "growing-like" animation from the center.
Is there anything related to this on planning to be added? I really wish I could stop the counter from using the default from top-right to bottom-left animation.

How it currently looks:
![image](https://user-images.githubusercontent.com/17398632/200662680-5334702f-e172-456f-b292-c6963f4c0deb.png)

How it used to look:
![image](https://user-images.githubusercontent.com/17398632/200665061-a68daad8-22db-4e25-9c01-c3add68eb1b9.png)

**Comment by Doksha** [comment]
seems more like a bug/issue that needs to be fixed imo (unless it has been stated elsewhere to be intended)

**Comment by peppy** [comment, marked as answer]
The components in legacy skins are being implemented to match stable 1:1, which is what was changed here.

We will add more options for customising the visual appearance of skinnable UI in the future, so stay tuned for that.
